.gitignore

```diff
@@ -9,3 +9,4 @@ Berksfile.lock
 .librarian
 Puppetfile.lock
 .kitchen.local.yml
+binstubs
```

Comment:
It's a typical place where `bundle install` puts binstubs (the main alternative to `bin`, which we use here).